packages/create/src/core.js

```diff
@@ -10,6 +10,8 @@ import deepmerge from 'deepmerge';
 
 import Generator from './Generator.js';
 
+import { cliOptions } from './helpers.js';
+
 const execAsync = util.promisify(exec);
```

Comment:
core should stay pretty clear of specific logic (e.g we may want to extract it at some point)

probably best to move it to `create.js` and then pass it on like so
```
    const module = await import(fullGeneratorPath);
    await executeMixinGenerator(module.default, { cliOptions }); // you will need to make that option as well
```

that way should be able to access it and pass it along inside of the generator mixins

Reply:
I tried this, but I couldnt get it to work nicely like this, I instead set the cliOptions as a property on the Generator class